### ISSUE TITLE
Impose "six" version requirement

### ISSUE THREAD

**Issue by svdvonde** [hickford/MechanicalSoup#9]
When I tried using the library in a Django production environment, it did not work with the (presumably) already-present version 1.3.0 of "six". It gave me some "could not import module urllib" error. When I set a version restriction on six in my own application (1.9.0), I could use mechanicalsoup perfectly fine. So perhaps the standard library needs to specify a minimal version of six that works, in order to prevent these situations (e.g. >= 1.9.0)

**Comment by hickford** [comment]
Thanks!

Out of curiosity how are you using MechanicalSoup? (If you want to share)

**Comment by svdvonde** [comment]
It's not a business production environment (if that's what I made it sound like), but it was the production environment of a website I'm building, hosted on Red Hat OpenShift (which apparently doesn't have six version >= 1.4.0).

The student-administration systems of the university I'm at are not very good for contacting students. For example, it is impossible for student organisations (like mine, for example) to contact specific groups of students (it's either everyone, or no one). Even the faculties can't contact specific groups of people (e.g. "all students of computer science"). So I'm building a scraper to create my own mailing lists. To log in to (and navigate) the online platform, I needed a simple tool ("not a massive library") so that I didn't have to worry about writing everything myself in the requests library. Since I couldn't find any popular libraries that do what I want, and do it for python3, I found this and used it. it's pretty neat.
